.circleci/config.yml

```diff
@@ -1,3303 +0,0 @@
-version: 2.1
```

Comment:
Will re-instate this file when the PR is otherwise good to go. With this deleted, I can debug GitHub Actions CI without triggering a full CircleCI build with every commit.